manifests/p/Python/Python/3/11/3.11.6/Python.Python.3.11.locale.en-US.yaml

```
@@ -0,0 +1,35 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Python.Python.3.11
+PackageVersion: 3.11.6
+PackageLocale: en-US
+Publisher: Python Software Foundation
+PublisherUrl: https://www.python.org/
+PublisherSupportUrl: https://www.python.org/about/help/
+PrivacyUrl: https://www.python.org/privacy/
+Author: Python Software Foundation
+PackageName: Python 3.11
+PackageUrl: https://www.python.org/
+License: PSF-2.0
+LicenseUrl: https://docs.python.org/3/license.html
+Copyright: |-
+  Copyright (c) 2001-2023 Python Software Foundation. All Rights Reserved.
+  Copyright (c) 2000 BeOpen.com. All Rights Reserved.
+  Copyright (c) 1995-2001 Corporation for National Research Initiatives. All Rights Reserved.
+  Copyright (c) 1991-1995 Stichting Mathematisch Centrum, Amsterdam. All Rights Reserved.
+CopyrightUrl: https://www.python.org/about/legal/
+ShortDescription: Python is a programming language that lets you work more quickly and integrate your systems more effectively.
+Moniker: python3

```

```suggestion
Moniker: python3.11
```

Since 3.12.0 stable version has now been added to WinGet repo, the `python3` moniker should be shifted to 3.12 manifests